packages/ui-kit/src/components/DataGrid/Drawer.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames'
+import React, { useRef } from 'react'
+
+import componentStyles from './Drawer.module.scss'
+import { DrawerProps } from './Drawer.types'
+import { ArrowDownIcon, CloseIcon, CopyIcon } from './icons'
+import { getDisplayValue } from './utils'
+
+// TODO: abstract Drawer to a generic component?
```

Comment:
Can you please clean up `DataGrid` folder, because currently it has files related to `Drawer`, `Loader` and `icons`.  It's a bit messy.
It can even be folders inside the `DataGrid` folder:
```
DataGrid/
  Drawer/
  Loader/
  Icons/
  ...
  {DataGrid files}
```